docker-compose.build.yml

```diff
@@ -5,40 +5,40 @@ services:
     build:
       context: 'server/'
       dockerfile: 'Dockerfile.packages'
-    image: liberoadmin/reviewer_server_packages:${IMAGE_TAG}
-    networks: 
+    image: libero/reviewer_server_packages:${IMAGE_TAG}
```

Comment:
Does this mean we were able to obtain the `libero` namespace on Docker Hub?

Reply:
no, the current convention is to use `libero/` locally and `liberoadmin/` when pushing - but we may need to register a new final value like `liberopub/` becuase requests for that namespace have been ignored